schemas/src/registry_schemas/schemas/filing.json

```diff
@@ -143,12 +143,28 @@
     ],
     "properties": {
         "filing": {
+            "properties": {
+                "filingType": {
+                    "enum": ["incorporation"]
+                }
+            },
             "allOf": [
                 {
                     "$ref": "#/definitions/filing_header"
                 },
                 {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business"
+                    "if": {
+                        "properties": {
+                            "filingType": {
+                                "not": {
+                                  "const": "incorporation"
+                                }
+                            }
```

Comment:
Are we not able to nest this condition a bit more to get to filing.header.name, rather than adding the new property filing.filingType? Adding another field here, which basically duplicates the same thing as in header, feels like too much.

Also (if you keep this new property) we should call it the same as header, so "incorporationApplication" rather than "incorporation".

Reply:
I looked into the options here, as i would also prefer not having an additional property. I will keep looking into it and find a way! 🤞

Reply:
Do you have any suggestions of how to approach this without using an additional property, i'm struggling finding a way to reference a specific property from another local reference?

Reply:
I had thought it was along the lines of:
```
"if": {
   "properties": {
      "header": {
          "properties": {
               "name": {
                   "not": {
                      "const": "incorporation"                             
```

However I can see no actual evidence of this. Let me just use a schema validator to check.

Reply:
I've been trying variations of this, i'll keep at it 👍

Reply:
Yep just did a simple check of the syntax and it works! FYI you can use https://www.jsonschemavalidator.net to check syntax with simplified examples, that's what I use.

Reply:
> Yep just did a simple check of the syntax and it works! FYI you can use https://www.jsonschemavalidator.net to check syntax with simplified examples, that's what I use.

(using "incorporationApplication", not "incorporation" as the constant).

Reply:
You're awesome, i was overthinking this. Has been updated!